gums-service/src/main/java/org/opensciencegrid/gums/service/JSONMapper.java

```diff
@@ -105,6 +109,49 @@ private void doVersion(HttpServletRequest request,
     }
 
 
+    private void doGenerateOsgUserVoMap(HttpServletRequest request,
+                          HttpServletResponse response)
+        throws ServletException, IOException
+    {
+        Configuration configuration = null;
+        try
+        {
+            configuration = gums.getConfiguration();
```

Comment:
This was a mistake - I was following the pattern of the other doX functions here and didn't notice it was unused.  I'll remove it.